Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/TouchHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TouchHandler : MonoBehaviour, IMixedRealityTouchHandler
+{
+
+    #region Event handlers
+    public TouchEvent OnTouchStarted = new TouchEvent();
```

Comment:
>TouchEvent [](start = 11, length = 10)

What about having TouchEvent contain HandTrackingInputEventData directly instead of TouchEventData?

Reply:
I thought about this, but HandTrackingInputEventData has fields like "InputData" which is hard to interpret. I think TouchPosition is more clearly communicating the underlying data. Think it will be eaiser for people to understand.

Reply:
I thought about this, but HandTrackingInputEventData has fields like "InputData" which is hard to interpret. I think TouchPosition is more clearly communicating the underlying data. Think it will be eaiser for people to understand.

Reply:
I thought about it more and decided to go with your suggestion, since I noticed PointerHandler uses a similar pattern.